api/libproject/devstart.rb

```diff
@@ -2402,8 +2402,10 @@ def deploy(cmd_name, args)
   common = Common.new
   common.status "Running database migrations..."
   ENV.update(read_db_vars(gcc))
-  migrate_database(op.opts.dry_run)
-  load_config(ctx.project, op.opts.dry_run)
+  CloudSqlProxyContext.new(gcc.project).run do
+    migrate_database(op.opts.dry_run)
```

Comment:
if `migrate_database()` always needs the proxy (looks like yes) then we should move this inside the function

Reply:
Done.